Error handling seems to be broken

When I have a syntax error in my scss the plugin dies with a NPE:

```
Caused by: java.lang.NullPointerException
	at java.io.Writer.write(Writer.java:157)
	at wrm.CompilationMojo.writeContentToFile(CompilationMojo.java:275)
	at wrm.CompilationMojo.processFile(CompilationMojo.java:261)
	at wrm.CompilationMojo.access$000(CompilationMojo.java:32)
	at wrm.CompilationMojo$1.visitFile(CompilationMojo.java:170)
	at wrm.CompilationMojo$1.visitFile(CompilationMojo.java:165)
	at java.nio.file.Files.walkFileTree(Files.java:2670)
	at java.nio.file.Files.walkFileTree(Files.java:2742)
	at wrm.CompilationMojo.execute(CompilationMojo.java:165)
	at org.apache.maven.plugin.DefaultBuildPluginManager.executeMojo(DefaultBuildPluginManager.java:134)
	... 21 more
```

`out.getCss()` seems to be `null` but no `CompilationException` is thrown

Complete Build:

Command:
```bash
mvn com.github.warmuuh:libsass-maven-plugin:0.2.1-libsass_3.3.3:compile
```

Output:
```mvn
[INFO] Scanning for projects...
[INFO]                                                                         
[INFO] ------------------------------------------------------------------------
[INFO] Building <project-name> 4.9-SNAPSHOT
[INFO] ------------------------------------------------------------------------
[INFO] 
[INFO] --- libsass-maven-plugin:0.2.1-libsass_3.3.3:compile (default-cli) @ shared-si ---
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time: 1.438 s
[INFO] Finished at: 2016-02-18T12:03:45+01:00
[INFO] Final Memory: 8M/208M
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal com.github.warmuuh:libsass-maven-plugin:0.2.1-libsass_3.3.3:compile (default-cli) on project <project>: Execution default-cli of goal com.github.warmuuh:libsass-maven-plugin:0.2.1-libsass_3.3.3:compile failed. NullPointerException -> [Help 1]
[ERROR] 
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR] 
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/PluginExecutionException

```

--- a/src/main/java/wrm/CompilationMojo.java
+++ b/src/main/java/wrm/CompilationMojo.java
@@ -256,6 +256,14 @@ public class CompilationMojo extends AbstractMojo {
 			return false;
 		}
 
+		if (out.getErrorStatus() != 0) {
+			getLog().error(out.getErrorMessage());
+			getLog().debug("ErrorText: " + out.getErrorText());
+			getLog().debug("ErrorFile: " + out.getErrorFile());
+			getLog().debug("ErrorJson: " + out.getErrorJson());
+			return false;
+		}
+
 		getLog().debug("Compilation finished.");
 
 		writeContentToFile(outputFilePath, out.getCss());

